modules/widgets/milestone/milestone.php

```diff
@@ -63,7 +63,7 @@ function __construct() {
 
 		add_action( 'wp_enqueue_scripts', array( __class__, 'enqueue_template' ) );
 		add_action( 'admin_enqueue_scripts', array( __class__, 'enqueue_admin' ) );
-		add_action( 'wp_footer', array( $this, 'localize_script' ) );
```

Comment:
Are you sure you had the right files uploaded?  This is not calling it statically, and I don't see any errors in master.